src/org/simple/diary/SimpleDiary.java

```diff
@@ -13,15 +13,22 @@
 
     public void setTitle(String title) {
         // TODO Implement...
+        //settitle에는 넣으면 []속에 들어가고
+        this.title = "["+title+"]";
     }
 
     public void setContent(String content) {
         // TODO Implement...
+        //content면 그냥 내용을 부르는 함수 이 함수는 SimpleDiary에 있다.
+        this.content = content;
     }
 
     @Override
     public String toString() {
+        //원래 문자열 리턴해 주는 앤데 기능을 바꿔서 이렇게??
+        //문자열을 다시 합쳐서 하나의 형태로 만드는 아이
         // TODO Implement...
-        return "";
+        String all = this.title.concat(this.content);
+        return all;
```

Comment:
어 이건 그냥 문자열 연산을 하면 좋을 것 같아요!
```suggestion
        return title + content;
```